redbot/core/bot.py

```diff
@@ -413,11 +419,65 @@ def colour(self) -> NoReturn:
 
         packages = []
 
-        if cli_flags.no_cogs is False:
-            packages.extend(await self._config.packages())
-
-        if cli_flags.load_cogs:
-            packages.extend(cli_flags.load_cogs)
+        last_system_info = await self._config.last_system_info()
+
+        async def notify_owners(content: str) -> None:
+            destinations = await self.get_owner_notification_destinations()
+            for destination in destinations:
+                prefixes = await self.get_valid_prefixes(getattr(destination, "guild", None))
+                prefix = prefixes[0]
+                await self.send_to_owners(content.format(prefix=prefix))
```

Comment:
```suggestion
                await destination.send(content.format(prefix=prefix))
```

Reply:
+ each of these sends needs to not prevent any of the others if they fail with forbidden due to closed dms or something